src/feature/Employee-profile/Employee-profile.module.css

```diff
@@ -0,0 +1,44 @@
+.main {
+  width: 80vw;
+}
+.employee__profile__section1 {
+  border-radius: var(--border-radius);
+  padding: 10px;
+  box-shadow: var(--box-shadow);
+  transition: box-shadow 0.3s;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
I see that you're using  `justify-content` and `align-items`. Will this work without a `display` set to flex?